Sources/NFCPassportReader/NFCPassportModel.swift

```diff
@@ -69,6 +68,23 @@ public class NFCPassportModel {
         return telephone
     }()
 
+    public private(set) lazy var personalNumber: String = {
+        guard let dg11 = dataGroupsRead[.DG11] as? DataGroup11,
+              let personalNumber = dg11.personalNumber else {
+                  if var dg1PersonalNumber = passportDataElements?["53"] {
+                      if dg1PersonalNumber.count >= 14 {
+                          dg1PersonalNumber = String(dg1PersonalNumber.dropFirst())
```

Comment:
This does not work on my Swedish passport, leaves the format YYYMMDDXXXX. Could this also be an issue for long document numbers where the rest of the document number shows up in the optional data field?